src/OrchardCore.Modules/OrchardCore.Media/Filters/MediaFilters.cs

```diff
@@ -76,4 +77,27 @@ public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments argu
             return new ValueTask<FluidValue>(new StringValue(url));
         }
     }
+
+    public class AppendVersionFilter : ILiquidFilter
```

Comment:
Because it doesn't use the asset prefix anymore, it can be a more generic helper, so on my side i moved it and register it in `OC.DisplayManagement.Liquid`.